src/main/kotlin/org/jitsi/nlj/RtpSenderImpl.kt

```diff
@@ -72,7 +72,7 @@ class RtpSenderImpl(
     private val outgoingRtpRoot: Node
     private val outgoingRtxRoot: Node
     private val outgoingRtcpRoot: Node
-    private val incomingPacketQueue = PacketInfoQueue(id, executor, this::processPacket)
+    private val incomingPacketQueue = PacketInfoQueue("rtp-sender-incoming-packet-queue", executor, this::processPacket)
```

Comment:
do we need the name to be the same for all for stats to work or something?  i often find myself wanting to print things using a unique id to track flows.

Reply:
Yes, if they have the same name they now get aggregated in the same QueueStatistics